vite.config.js

```diff
@@ -39,6 +39,10 @@ export default defineConfig(() => {
       },
       extensions: ['.mjs', '.js', '.ts', '.jsx', '.tsx', '.json', '.vue']
     },
+    server: {
+      host: true,
+      port: 8080
```

Comment:
Before:
![before](https://github.com/bcgov/business-create-ui/assets/122301442/0158203b-6b70-4409-b3a5-7a728d374fed)

After:
![after](https://github.com/bcgov/business-create-ui/assets/122301442/5eeba5b5-5df5-486c-8dc4-0d66048e85dc)

Now, the links will work the exact same as before to not confuse other Devs. For example: http://localhost:8080/incorporation-people-roles?id=TAEUG9DPGd

Reply:
stealing this

Reply:
All yours Travis 😎

Reply:
👍 👍 👍